src/shared/components/floating-button/floating-button.css.ts

```diff
@@ -0,0 +1,71 @@
+import { style } from "@vanilla-extract/css";
+
+import { themeVars } from "@shared/styles/theme.css";
+
+const baseButton = style({
+  position: "absolute",
+  right: "2.9rem",
+  display: "flex",
+  width: "4.2rem",
+  height: "4.2rem",
+  padding: "1.2rem",
+  alignItems: "center",
+  justifyContent: "center",
+  gap: "1.2rem",
+  borderRadius: "20.5px",
+  aspectRatio: "1/1",
+  cursor: "pointer",
+  border: "none",
+  transition: "opacity 0.1s ease",
+  boxShadow: "2px 2px 8px 0px rgba(0, 0, 0, 0.15)",
+  selectors: {
+    "&:active": {
+      opacity: 0.7,
+    },
+  },
+});
+
+export const floatingButtonTop = style([
+  baseButton,
+  {
+    bottom: "15.2rem",
+    background: themeVars.color.grayscale9,
+    color: themeVars.color.grayscale1,
+  },
+]);
+
+export const floatingButtonHelp = style([
+  baseButton,
+  {
+    bottom: "9.2rem",
```

Comment:
```suggestion
    bottom: "6.2rem",
```
이 부분도 이 값이 맞는것 같아요!

<img width="319" height="730" alt="Image" src="https://github.com/user-attachments/assets/482a6d55-e076-495c-8aea-03229010064a" />